tests/Unit/Pypp/PyppFundamentals.hpp

```diff
@@ -221,7 +221,8 @@ struct FromPyObject<long, std::nullptr_t> {
 #if PY_MAJOR_VERSION == 2 && PY_MINOR_VERSION == 7
     } else if (not PyInt_Check(t) and not PyLong_Check(t)) {
 #elif PY_MAJOR_VERSION == 3
-    } else if (not PyLong_Check(t)) {
+      // clang-tidy: hicpp-signed-bitwise
+    } else if (not PyLong_Check(t)) { // NOLINT
```

Comment:
why not use `PyLong_Check(t) == 0`?

Reply:
Why change it? I was just trying to silence a clang-tidy warning and it would have to be done for every other check in this file.

Reply:
Our thinking was just that treating the int return type more transparently is marginally better code style than needing to silence the linter. If it's a decision that would require numerous changes I am not personally that concerned about it (though the other Caltech reviewers should feel free to disagree).

Reply:
(I do not disagree)